node/cmd/guardiand/node.go

```diff
@@ -469,6 +469,16 @@ func runNode(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
+	// Determine execution mode
+	var env common.Environment
+	if *unsafeDevMode {
+		env = common.UnsafeDevNet
+	} else if *testnetMode {
+		env = common.TestNet
+	} else {
+		env = common.MainNet
+	}
```

Comment:
Might be nice to have a log somewhere for what env and network is being used if it isn't anywhere already.